### PR TITLE
Handbrake 1.1.1

### PR DESCRIPTION
**Pull request by kenmays**
Adds handbrake 1.1.1 recipe/patchset to Haikuports.
Tested on Haiku hrev52225 x86_64.

### CODE REVIEW

**Comment by extrowerk** [comment]
Please remove the 1.0.7 patchset.

**Comment by extrowerk** [comment]
Your recipe got a trailing whitespace somewhere. Please, remove it.

File: media-video/handbrake/handbrake-1.1.1.recipe [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+Convert from many common multimedia file formats, including unprotected \
+DVD or BluRay sources to a handful of modern output file formats."
+HOMEPAGE="http://handbrake.fr/"
+LICENSE="GNU GPL v2"
+COPYRIGHT="2003-2018 HandBrake Team"
+REVISION="1"
+SOURCE_URI="http://download.handbrake.fr/releases/$portVersion/HandBrake-$portVersion-source.tar.bz2"
+CHECKSUM_SHA256="e3390c5fd901fb06d72e29c62a63d373d5fb5b3467295d114d815ae7b78a9d7a"
+SOURCE_DIR="HandBrake-$portVersion"
+#PATCHES="handbrake-$portVersion.patchset"
```
Comment (extrowerk) [inline media-video/handbrake/handbrake-1.1.1.recipe]: Isn't it needed?

File: media-video/handbrake/handbrake-1.1.1.recipe [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+PROVIDES="
+	handbrake$secondaryArchSuffix = $portVersion
+	cmd:HandBrakeCLI
+#	app:HandBrake
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:liba52$secondaryArchSuffix
+	lib:libass$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
```
Comment (extrowerk) [inline media-video/handbrake/handbrake-1.1.1.recipe]: whitespace

File: media-video/handbrake/handbrake-1.1.1.recipe [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+	lib:libvpx$secondaryArchSuffix
+	lib:libx264$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:liba52$secondaryArchSuffix
+	devel:libass$secondaryArchSuffix
```
Comment (extrowerk) [inline media-video/handbrake/handbrake-1.1.1.recipe]: whitespace

File: media-video/handbrake/handbrake-1.1.1.recipe [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+	lib:libx264$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:liba52$secondaryArchSuffix
+	devel:libass$secondaryArchSuffix 
+	devel:libavcodec$secondaryArchSuffix
```
Comment (extrowerk) [inline media-video/handbrake/handbrake-1.1.1.recipe]: whitespace

File: media-video/handbrake/handbrake-1.1.1.recipe [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:liba52$secondaryArchSuffix
+	devel:libass$secondaryArchSuffix 
+	devel:libavcodec$secondaryArchSuffix 
+	devel:libbluray$secondaryArchSuffix
```
Comment (extrowerk) [inline media-video/handbrake/handbrake-1.1.1.recipe]: whitespace

File: media-video/handbrake/handbrake-1.1.1.recipe [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+{
+	sed -i 's/MODULES += contrib\/.*//g' make/include/main.defs
+}
+
+BUILD()
+{
+	./configure --prefix=$prefix --force --disable-x265
+	cd build
+	make
+	cd ../haiku
```
Comment (extrowerk) [inline media-video/handbrake/handbrake-1.1.1.recipe]: required?

**Comment by kenmays** [comment]
We need to keep my working copy of handbrake 1.0.7. Why not merge it?

**Comment by extrowerk** [comment]
If this superseeds the 1.0.7 PR, then close that, and clean this up.

-- Z.

2018.08.16. dátummal, 17:05 időpontban kenmays <notifications@github.com> írta:

**Pull request closed without merging**